core/src/main/java/org/eclipse/hono/util/EventBusMessage.java

```diff
@@ -535,6 +537,27 @@ public EventBusMessage setStringProperty(final String name, final Message msg) {
         }
     }
 
+    /**
+     * Sets a {@code SpanContext} on this message.
```

Comment:
I think you can come up with a more meaningful description, can't you? The fact that this is the setter for a `SpanContext` is obvious. However, it might be interesting to know what this is good for ...

Reply:
ok, I suggest this:
````
* Sets a {@code SpanContext} on this message.
* This may be the {@code SpanContext} associated with the AMQP message for which this {@code EventBusMessage} was created.  
* When tracing any operations on this {@code EventBusMessage}, this {@code SpanContext} can be used as parent for the
* created <em>OpenTracing</em> spans.
* <p>
* Note: this {@code SpanContext} instance will not get serialized when sending
* the message over the vert.x event bus!
````

Reply:
what about
```
* Sets an <em>OpenTracing</em> {@code SpanContext} on this message.
* <p>
* This may be the tracing context associated with the (request) message for which this {@code EventBusMessage} was created.
* The span context should be used as the parent context of each OpenTracing {@code Span} that is created as part of processing this event bus message.
* <p>
* Note: the span context instance will not get serialized when sending
* this event bus message over the vert.x event bus!
```
?

Reply:
Good, I've amended the commit accordingly.